scripts/deployment/deploy_ec2_worker.sh

```
@@ -42,9 +42,17 @@ aws s3 cp s3://cloudcv-secrets/evalai/${ENVIRONMENT}/docker_${ENVIRONMENT}.env .
+    WORKER_COMPOSE_SERVICE="worker_py3_9"
+    if [ -n "${WORKER_PYTHON_VERSION}" ]; then
+        case "${WORKER_PYTHON_VERSION}" in
+            3.7) WORKER_COMPOSE_SERVICE="worker_py3_7" ;;
+            3.8) WORKER_COMPOSE_SERVICE="worker_py3_8" ;;
+            3.9) WORKER_COMPOSE_SERVICE="worker_py3_9" ;;
+        esac
+    fi
     # Step 8: Pulling worker Docker image
     echo "Step 8/10: Pulling worker Docker image"
-    docker compose -f docker-compose-${ENVIRONMENT}.yml pull worker
+    docker compose -f docker-compose-${ENVIRONMENT}.yml pull ${WORKER_COMPOSE_SERVICE}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)